packages/html/src/plugins/responsive.ts

```diff
@@ -26,32 +27,33 @@ export function responsive({steps}:{steps?: number | number[]}={}): Plugin{
  * @param element {HTMLImageElement} The image element
  * @param responsiveImage {CloudinaryImage}
  * @param htmlPluginState {HtmlPluginState} holds cleanup callbacks and event subscriptions
+ * @param analyticsOptions {AnalyticsOptions} analytics options for the url to be created
  */
-function responsivePlugin(steps?: number | number[], element?:HTMLImageElement, responsiveImage?: CloudinaryImage, htmlPluginState?: HtmlPluginState): Promise<void | string> | boolean {
+function responsivePlugin(steps?: number | number[], element?: HTMLImageElement, responsiveImage?: CloudinaryImage, htmlPluginState?: HtmlPluginState, analyticsOptions?: AnalyticsOptions): Promise<void | string> | boolean {
 
-  if(!isBrowser()) return true;
+    if (!isBrowser()) return true;
 
-  if(!isImage(element)) return;
```

Comment:
Here and below it looks like some auto formatting whitespace changes.

Reply:
right i will fix